Adafruit_DRV2605.cpp

```diff
@@ -51,14 +51,37 @@ Adafruit_DRV2605::Adafruit_DRV2605() {
 /*                           PUBLIC FUNCTIONS                             */
 /*========================================================================*/
 
+/**************************************************************************/
+/*!
+  @brief Setup HW using the default Wire
+  @return Return value from init()
+*/
+/**************************************************************************/
+boolean Adafruit_DRV2605::begin() {
+  _wire = &Wire;
+  return init();
+}
+
+/**************************************************************************/
+/*!
+  @brief Setup HW using a different Wire
+  @param theWire Pointer to a TwoWire object
+  @return Return value from init()
+*/
+/**************************************************************************/
+boolean Adafruit_DRV2605::begin(TwoWire *theWire) {
```

Comment:
and having the argument `theWire=&Wire` (default set to built in Wire object)
https://www.programiz.com/cpp-programming/default-argument